README.md

```diff
@@ -621,6 +622,39 @@ api.PathPart = "users/abc";
 await api.GetAsync();
 ```
 
+HTTP Request Message Properties
+-----------------
+
+In very specific cases (i. e. you use custom `DelegatingHandler`), it might be useful to pass any object reference into the handler. In such case `HttpRequestMessage.Properties` can be used.
```

Comment:
`DelegatingHandler` isn't used elsewhere in the README. Can we refer to `RequestModifier` and/or `HttpClientHandler` (since DelegatingHandler is a subclass of HttpClientHandler)?

Reply:
`DelegatingHandler` is subclass of `HttpMessageHandler`

Reply:
So it is - I got my hierarchy muddled up.